exp/api/v1beta1/types.go

```diff
@@ -276,5 +276,5 @@ type UpdateConfig struct {
 	// +optional
 	// +kubebuilder:validation:Maximum=100
 	// +kubebuilder:validation:Minimum=1
-	MaxUnavailablePercentage *int `json:"maxUnavailablePrecentage,omitempty"`
+	MaxUnavailablePercentage *int `json:"maxUnavailablePercentage,omitempty"`
```

Comment:
Is this technically a breaking change? If someone used this with the incorrect spelling and uses the new version it wouldn't recognise that setting. 

I'm think that we might have to introduce an intentionally misspelled version deprecate this one and redirect to the correctly spelled one. Unless people think that it's not necessary. 😁

Reply:
This is a breaking change [yes](https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/3467#issuecomment-1125979752).

Reply:
Right, I didn't read the previous comments. 🤦 

Err... so can we merge this now? Since we had 2.0.0 sort of..? @richardcase ?

Reply:
I also thought did we need a new struct field to force conversion, but the fuzzing tests would've picked this up.

I agree, as we are bumping the API version this breaking change is ok to go into 2.0.0. So

/lgtm